src/Transformers/ModelTransformer.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace Spatie\LaravelTypeScriptTransformer\Transformers;
+
+use Spatie\TypeScriptTransformer\Structures\TransformedType;
+
+use DB;
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\Schema;
```

Comment:
I am uncertain if the use of these imports is appropriate for this library. I do not see any direct references to Illuminate in the composer.json.

FWIW while I've been programming a long time, I'm relatively new to PHP package development.